js/utility/createDurationElement.js

```diff
@@ -0,0 +1,7 @@
+export default function createDurationElement(duration) {
+  const taskCompleteDuration = document.createElement("div");
```

Comment:
inconsistent variable naming. `createElement` does not return a duration rather a DOM element.

// https://developer.mozilla.org/en-US/docs/Web/API/Document/createElement

```suggestion
  const taskCompleteDuration = document.createElement("div");
```